libnmstate/state.py

```diff
@@ -381,9 +381,21 @@ def merge_interfaces(self, other_state):
         This is a reverse recursive update operation.
         """
         other_state = State(other_state.state)
+        origin_other_state = other_state
+        other_state = State(origin_other_state.state)
```

Comment:
I don't understand why is this needed. You are creating a copy of `other_state` twice.

Reply:
Thank you!